mis_builder/models/mis_report_instance.py

```diff
@@ -463,6 +463,9 @@ def _default_company_id(self):
     analytic_account_id = fields.Many2one(
         comodel_name='account.analytic.account', string='Analytic Account',
         oldname='account_analytic_id')
+    analytic_account_filter_id = fields.Many2one(
+        comodel_name='account.analytic.account',
+        string='Analytic Account Filter')
```

Comment:
Unfortunately adding a field sounds like a blocking issue? It has security implication (user needs write access on the template). And it probably break when several users preview the same report?

Reply:
It is executing a write on the field analytic_account_filter_id so, yes @sbidoul you are right.